test/slider-test.jsx

```diff
@@ -9,6 +9,14 @@ import has from 'has';
 import Slider from '../src/Slider';
 import { KEYS } from '../lib/constants/SliderConstants';
 
+const { WITH_DOM } = process.env;
+
+function skipWithDom() {
```

Comment:
fair enough :-p feel free to rename it.

I mainly wanted the default to be node, and "test with a dom" to be the special case.